tests/test_models/test_engine/test_db_storage.py

```diff
@@ -86,3 +86,51 @@ def test_new(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_get_0_arg(self):
+        """Test that get no exist properly saves objects to file.json"""
+        with self.assertRaises(TypeError):
+            models.storage.get()
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_get_1_arg(self):
+        """Test that get no exist properly saves objects to file.json"""
```

Comment:
This comment is the same than before, verify and fix according to the specific case